README.md

```diff
@@ -34,18 +34,22 @@ Nulecule specification enables complex applications to be defined, packaged and
 
 ## Deployment User Experience
 
-Here's an example using the [atomicapp reference implementation](https://github.com/projectatomic/atomicapp) with a kubernetes provider.
+The Nulecule specification has been implemented in the [atomicapp reference implementation](https://github.com/projectatomic/atomicapp).  Atomicapp currently supports docker containers and kubernetes and docker orchestration providers.  The [atomic command](https://github.com/projectatomic/atomic) is used to run the container that contains the Nulecule specification and the atomicapp implementation.
 
```

Comment:
How about:

Atomicapp currently supports docker containers and two orchestration providers, kubernetes and base docker.

I think the word containers is important to different it from rkt or docker as an "orchestrator"